opensearch-operator/examples/opensearch-cluster-custom-admin-user.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: opensearch.opster.io/v1
```

Comment:
Please add a comment as first line explaining what this file is an example for and when to use it